### PR TITLE
modify auto-api template

### PR DESCRIPTION
**Pull request by WenjiaoYue**
## Type of Change

documentation
API changed or not
no

## Description

modify auto-api template

## Expected Behavior & Potential Risk

the content of the file can be displayed more reasonably and clearly

## How has this PR been tested?

click docs link

## Dependency Change?

no

### CODE REVIEW

**Review by NeoZhangJianyu — APPROVED** [review]
Approve.

**Comment by chensuyue** [comment]
@NeoZhangJianyu @WenjiaoYue can we modify the API structure here as what we discussed by email?

File: neural_compressor/api_doc_example.py [hunk 1]
```diff
@@ -0,0 +1,208 @@
+"""
```
Comment (chensuyue) [inline neural_compressor/api_doc_example.py:1]: Please move the example into another place.

**Review by chensuyue — APPROVED** [review]

**Pull request merged**